templates/default/gcc_modulefile.erb

```diff
@@ -0,0 +1,44 @@
+#%Module1.0
+
+## Required internal variables
+set name     gcc
+set version  <%= node['cfncluster']['armpl']['gcc']['version'] %>.0
```

Comment:
I followed up with Yulei offline about this. We hardcode this patch number here and in the installer URL. Ideally we could add another parameter in the default attributes for this, because if we did ever want to use a patch update other than `.0` it seems likely that this change would be forgotten.

Reply:
Done, use major_minor_version and patches_version